DemoApp/Sources/Components/Feedback/DemoFeedbackView.swift

```diff
@@ -0,0 +1,155 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamVideo
+import StreamVideoSwiftUI
+import SwiftUI
+
+@available(iOS 15.0, *)
```

Comment:
why is it only iOS 15+?

Reply:
Because we are using `@Environment(\.dismiss)`

Reply:
you can also use presentationMode from the environment instead

Reply:
I'm more inclined to use the custom dismiss closure as it also serves to reset the internal state.